pytorch/src/main/java/org/bytedeco/pytorch/presets/torch_cuda.java

```diff
@@ -72,15 +94,8 @@ public void map(InfoMap infoMap) {
         torch.sharedMap(infoMap);
 
         infoMap
-            .put(new Info("basic/containers").cppTypes("c10::optional"))
-
             .put(new Info().enumerate().friendly())
             .put(new Info().javaText("import org.bytedeco.pytorch.*;"))
-            .put(new Info().javaText("import org.bytedeco.pytorch.cuda.functions.*;"))
-            .put(new Info().javaText("import org.bytedeco.pytorch.Error;"))
-            .put(new Info().javaText("import org.bytedeco.pytorch.global.torch.DeviceType;"))
-            .put(new Info().javaText("import org.bytedeco.pytorch.global.torch.ScalarType;"))
-            .put(new Info().javaText("import org.bytedeco.pytorch.global.torch.MemoryFormat;"))
             .put(new Info().javaText("import org.bytedeco.pytorch.Allocator;"))
```

Comment:
And this one too? Or is that required to override a conflict somewhere...

Reply:
Yes, this one is useful to disambiguate between `org.bytedeco.pytorch.Allocator` and `org.bytedeco.javacpp.annotation.Allocator`.